validphys2/src/validphys/cuts/filters.yaml

```diff
@@ -388,3 +388,8 @@
   FNS: FONLL-C
   IC: True
   rule: "Q2 > 8"
+
+- dataset: H1_1JET_319GEV_290PB-1_DIF_PTQ2
+  reason: |
+    There are 8 Q^2 bins, however the grid for the 1st bin is unavailable.
+  rule: "q_sqr >= 8"
```

Comment:
Remember to change this to Q2 as well when you search-and-replace

Reply:
done